### PR TITLE
Ravinder | Test | Add an option in UI for putting custom voucher number for Purchase Debit note and Credit note.

### PR DESCRIPTION
**Pull request by ravinderwalkover**
https://app.clickup.com/t/b40g0a

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-1.scss [hunk 1]
```diff
@@ -3229,3 +3229,6 @@ td.mat-footer-cell {
     transform: none !important;
 }
 /* material css for ledger end*/
+.float-left {
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/assets/css/style-1.scss:3232]: @ravinderwalkover float-left is already a predefined class in bootstrap, we can remove this from here

**Review by aditi-tiwari17 — APPROVED** [review]

File: apps/web-giddh/src/app/material-ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html [hunk 2]
```diff
@@ -1541,6 +1552,28 @@ <h3 *ngIf="!openDropDown" (dblclick)="openHeaderDropDown()" class="d-inline-bloc
...
+                                                "
+                                                class="pd-r1 pb-1"
+                                            >
+                                                <label class="default-color">{{ commonLocaleData?.app_number }}</label>
+                                                <input
+                                                    matInput
+                                                    id="update-ledger-invoice-number"
+                                                    type="text"
+                                                    [placeholder]="
+                                                        voucherApiVersion === 2
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/material-ledger/components/update-ledger-entry-panel/update-ledger-entry-panel.component.html:1569]: This check is not required as the parent `td` will only be rendered when `voucherApiVersion` has value `2`

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please make the changes in old ledger as well

**Comment by ravinderwalkover** [comment]
> Please make the changes in old ledger as well

@prateek-giddh we will only use material ledger on PROD. I discussed with @shubhendra sir.

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**